sourcegit/watcher.py

```diff
@@ -32,16 +32,17 @@ def process_pr(self, msg):
         :return:
         """
         project_name = msg["msg"]["pullrequest"]["project"]["name"]
-
-        ps = PagureService(token=self.pagure_token)
-        project = ps.get_project(repo=project_name, namespace="rpms")
+        logger.info("new flag for PR for %s", project_name)
 
         try:
-            logger.info("new flag for PR for %s", project_name)
             source_git = package_mapping[project_name]["source-git"]
```

Comment:
As we discussed -- this can be fixed in the config-related PR.

Reply:
oki